src/qt/intro.cpp

```diff
@@ -255,11 +255,11 @@ void Intro::setStatus(int status, const QString &message, quint64 bytesAvailable
         if (bytesAvailable < m_required_space_gb * GB_BYTES) {
             freeString += " " + tr("(of %n GB needed)", "", m_required_space_gb);
             ui->freeSpace->setStyleSheet("QLabel { color: #800000 }");
-            ui->prune->setChecked(true);
+            if (!keep_prune) ui->prune->setChecked(true);
```

Comment:
In commit "gui: Let user to uncheck prune on low-space dirs" (9672ea841d7a66c765d734cfab57210baa4b074d)

Can you update the commit description to say behavior was before and after this change? It seems without this code the previous commit f07dd7c69ed7a5245f79662100c859286557b73b will be broken, because you added a new space check when the checkbox is toggled, so the box would automatically toggle itself back on after the check completed? All of this is very unclear to me.

Reply:
> It seems without this code the previous commit [f07dd7c](https://github.com/bitcoin/bitcoin/commit/f07dd7c69ed7a5245f79662100c859286557b73b) will be broken...

You are right. Fixed in the latest push.